reactNativeProject/src/redux/item/actions.js

```diff
@@ -0,0 +1,15 @@
+export const types = {
+  ADD_ITEM: "ADD_ITEM",
+  REMOVE_ITEM: "REMOVE_ITEM",
+  TOGGLE_ITEM_COMPLETED: "TOGGLE_ITEM_COMPLETED"
+};
+
+export const actionCreators = {
+  addItem: item => ({ type: types.ADD_ITEM, payload: item }),
+  onRemoveItem: index => {
```

Comment:
Unify styles. In `addItem` you are returning the object without a code block, and in this one you are defining a code block with a return inside.